packages/rocketchat-graphql/server/api.js

```diff
@@ -10,15 +10,25 @@ import cors from 'cors';
 
 import { executableSchema } from './schema';
 
-const subscriptionPort = 3100;
+const subscriptionPort = RocketChat.settings.get('Graphql_Subscription_Port') || 3100;
 
 // the Meteor GraphQL server is an Express server
 const graphQLServer = express();
 
-graphQLServer.use(cors());
+if (RocketChat.settings.get('Graphql_CORS')) {
```

Comment:
What happens if this setting is changed after the server has started? Does it require a restart of the server to have an effect?